content/en/blog/_posts/2024-12-11-Kubernetes-v1-32-Release.md

```diff
@@ -0,0 +1,301 @@
+
+## Release team
+
+Kubernetes is only possible with the support, commitment, and hard work of its community. 
+Each release team is made up of dedicated community volunteers who work together to build the many pieces that make up the Kubernetes releases you rely on. 
+This requires the specialized skills of people from all corners of our community, from the code itself to its documentation and project management.
+
+We would like to thank the entire [release team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.32/release-team.md) for the hours spent hard at work to deliver the Kubernetes v1.32 release to our community. 
+The Release Team's membership ranges from first-time shadows to returning team leads with experience forged over several release cycles. 
+A very special thanks goes out our release lead, Frederico Muñoz, for leading the release team so gracefully and handle any matter with the uttermost care, making sure this release was executed smoothly and efficiently.
+Last but not least a big thanks goes to all the release members - leads and shadows alike - and to the following SIGs for the terrific work and outcome achieved during these 14 weeks of release work:
+
+- [SIG Docs](https://github.com/kubernetes/community/tree/master/sig-docs) - for the fundamental support in docs and blog reviews and continous collaboration with release Comms and Docs;
+- [SIG k8s Infra](https://github.com/kubernetes/community/tree/master/sig-k8s-infra) and [SIG Testing](https://github.com/kubernetes/community/tree/master/sig-testing) - for the outstanding work in keeping the testing framework in check, along with all the infra components necessary;
+- [SIG Release](https://github.com/kubernetes/community/tree/master/sig-release) - for the incredible support provided throughout the orchestration of the entire release, addressing challenging issues in a graceful and timely manner;
+
+
+## Project velocity
+
+The CNCF K8s [DevStats project](https://k8s.devstats.cncf.io/d/11/companies-contributing-in-repository-groups?orgId=1&var-period=m&var-repogroup_name=All) aggregates a number of interesting data points related to the velocity of Kubernetes and various sub-projects. This includes everything from individual contributions to the number of companies that are contributing and is an illustration of the depth and breadth of effort that goes into evolving this ecosystem.
+
+In the v1.32 release cycle, which ran for 14 weeks (September 9th to December 11th), at the time of writing we verified contributions to Kubernetes from 99 different companies and 506 individuals.
+
+In the whole Cloud Native ecosystem we have 385 companies counting 2195 total contributors - in line with the [previous release](https://kubernetes.io/blog/2024/08/13/kubernetes-v1-31-release/#project-velocity) cycle! 
```

Comment:
As discussed separately, I'm making a bit more adjustments to the wording and contents to rightly highlight the data points and moving average.

```suggestion
In the v1.32 release cycle, which ran for 14 weeks (September 9th to December 11th), we saw contributions to Kubernetes from as many as 125 different companies and 559 individuals as of writing.

In the whole Cloud Native ecosystem, the figure goes up to 433 companies counting 2441 total contributors. This sees an increase of 7% more overall contributions compared to the [previous release](https://kubernetes.io/blog/2024/08/13/kubernetes-v1-31-release/#project-velocity) cycle, along with 14% increase in the number of companies involved, showcasing strong interest and community behind the Cloud Native projects.
```